contract_layout_category_hide_detail/tests/test_contract_layout_category_hide_detail.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2019 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests import common
+
+
+class TestContractLayoutCategoryHideDetail(common.SavepointCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestContractLayoutCategoryHideDetail, cls).setUpClass()
+
+    # @classmethod
```

Comment:
You can omit tests for now and put it in the ROADMAP as we are over-passed on budget.